src/components/segmentedControl/index.tsx

```diff
@@ -150,29 +153,30 @@ const SegmentedControl = (props: SegmentedControlProps) => {
     });
 
   return (
-    <View row center style={[styles.container, containerStyle, {borderRadius, backgroundColor}]}>
-      <Reanimated.View
-        style={[
-          styles.selectedSegment,
-          animatedStyle,
-          {borderColor: outlineColor, borderRadius, backgroundColor: activeBackgroundColor, borderWidth: outlineWidth}
-        ]}
-      />
-      {renderSegments()}
+    <View>
+      <View row center style={[styles.container, containerStyle, {borderRadius, backgroundColor}]}>
```

Comment:
I think it's important to pass the `containerStyle` to the actual container view. 
This way the user can pass things like `alignSelf: flex-start` for alignment and have overall have better control of the container style. 
Maybe we can have a `style` prop for the segment control instead. 
WDYT? 

Usually, I would have done a migration for it, but I think since it's a new component, it's safe enough to make the change.